src/main/java/seedu/address/model/person/LoanRecords.java

```diff
@@ -39,27 +55,33 @@ public void addLoan(Loan loan) {
      * @param endDate A valid end date.
      */
     public void addLoan(float amount, Date startDate, Date endDate) {
-        Loan loan = new Loan(amount, startDate, endDate);
+        Loan loan = new Loan(nextLoanId, amount, startDate, endDate);
```

Comment:
As LoanRecords are created by person, is it possible for two loans to two different people to have the same id?